lib/txt/ansi/login1.ans

```diff
@@ -1,4 +1,4 @@
-[?7h[255D[0m        [0m/    [0;32m'  [0;31m<[1;34m|   [0;31m<[1;34m|   [0m\  /     /[32m  "     '             ,   [0m\               [K
+[255D[0m        [0m/    [0;32m'  [0;31m<[1;34m|   [0;31m<[1;34m|   [0m\  /     /[32m  "     '             ,   [0m\               [K
```

Comment:
What's this change do?

Reply:
I'm not sure what that ANSI character was intended to be, but it was messed up and was just displaying `7h` at the top-left of that login ASCII graphic. So I just took it out, and it looks fine now as far as I can tell.